.sandbox/provision/lifecycle-asg/common.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+: ${MAX_RETRIES:=30}
+: ${VOLUME_SIZE:=10}
+: ${DEVICE_NAME:="/dev/xvdf"}
+: ${SANDBOX_NAME_TAG:="SandboxManaged-SandboxName"}
+: ${SANDBOX_ID_TAG:="SandboxManaged-SandboxID"}
+: ${SANDBOX_ASG_TAG:="sandbox-asg"}
+
+function fatal() {
+  echo "$@" >&2
+  exit 1
+}
+
+function get_volume_id() {
+    local volume
+    volume="$(aws ec2 describe-volumes --filters Name=tag:"$SANDBOX_ID_TAG",Values="$SANDBOX_ID")"
+    jq -cMr '.Volumes[0].VolumeId' <<< "$volume"
+}
+
+function create_volume_if_needed() {
+    local volume_id="$(get_volume_id)"
+    if [[ "$volume_id" -eq null ]]; then
```

Comment:
This is a bit away from normal. `null` is something specific to `jq`, can we filter that inside `get_volume_id` and print empty string instead?